extensions/canvasMedical/actions/createAppointment.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { ZodError } from 'zod'
+import {
+  type Fields,
+  FieldType,
+  type Action,
+  type DataPointDefinition,
+} from '@awell-health/extensions-core'
+import { Category } from '@awell-health/extensions-core'
+import { type settings } from '../settings'
+import { makeAPIClient } from '../client'
+import { fromZodError } from 'zod-validation-error'
+import { AxiosError } from 'axios'
+import { appointmentSchema } from '../validation/dto/appointment.zod'
+import type schemas from '../schemas'
+
+const fields = {
+  appointment_data: {
+    id: 'appointment_data',
+    label: 'Appointment data',
+    description: 'Appointment data',
+    type: FieldType.JSON,
+    jsonType: 'canvas_appointment',
```

Comment:
Let's remove all the jsonType fields, they're not needed or used in the Awell platform